src/Database/DatabaseUtil.php

```diff
@@ -622,15 +627,14 @@ public function composeWhereForQueryBuilder(QueryBuilder $queryBuilder, string $
                 if (empty($value)) {
                     $where = $queryBuilder->expr()->eq(1, 2);
                 } else {
-                    $where = $queryBuilder->expr()->in(
-                        $field,
-                        array_map(
-                            function ($val) {
-                                return '"'.addslashes(Controller::replaceInsertTags(trim($val), false)).'"';
-                            },
-                            $value
-                        )
+                    $where = $queryBuilder->expr()->in($field, $wildcard);
+                    $preparedValue = array_map(
+                        function ($val) {
+                            return addslashes(Controller::replaceInsertTags(trim($val), false));
+                        },
+                        $value
```

Comment:
Why did you remove the double quotes? What if your field is a string field?

Reply:
Double quotes are not possible, there happen some kind of shenenigans when doctrine gets an array with double quotes like [""2"",""3""] 
doctrine needs array for IN and NOT IN 
https://github.com/doctrine/dbal/blob/012569dfcc96478c061d3b529fecb5c8ac0405e5/src/Query/Expression/ExpressionBuilder.php#L281 

it should not be possible to add a string

Reply:
or we need convert a string into array before seting it as IN/NOT IN parameter